### DISCUSSION TITLE
[WASM][.NET 6] Default templates can't be debugged

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#8428] [General]
Creating a .NET6 repo as per [here ](https://platform.uno/docs/articles/get-started-winui3.html) creates a solution which can't be debugged, since e.g. breakpoints won't work.

I checked launchSettings.js and found that the "inspectUri" line is missing. However, even adding that line as per [here ](https://platform.uno/docs/articles/debugging-wasm.html?q=debug)does not help: the WASM project still is not debuggable.

- what needed to be done to make the standard .NET 6 template debuggable?
- why making it so hard for your users and not providing a template which would have debugging enabled by default?

Note: my custom .NET 6 UNO WASM app still debugs no problem on the same machine - not sure what the difference is.

What am I missing?

**Comment by jeromelaban** [comment]
Thanks. It is likely because the bootstrapper version is 3.2, and you need 3.3 for your net6-only installation, we'll make changes to template so it's upgraded properly.

**Comment by DierkDroth** [comment]
@jeromelaban I'm not sure about that. 

- first screenshot shows the packages of my custom app where debugging works -> good
- second screenshot shows the latest UNO bits applied to my custom app **where debugging no longer works** -> **bad**
In fact browser console shows crazy errors (attached)

Any idea what's going wrong?

[localhost-1648476533986.zip](https://github.com/unoplatform/uno/files/8363501/localhost-1648476533986.zip)

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/26179242/160415849-4cc707bd-8977-4548-84dc-fbb91b5ef23f.png">
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/26179242/160416534-b2c3b1fb-a80b-4fb5-8096-31fb48840ed4.png">

**Reply by jeromelaban** [reply]
Thanks. The log that you've sent is for an app using Uno.Wasm.Bootstrap 4.0.0, please use stable builds instead.
